go/vt/worker/split_clone_test.go

```diff
@@ -992,9 +992,11 @@ func TestSplitCloneV2_NoMasterAvailable(t *testing.T) {
 	expectBlpCheckpointCreationQueries(tc.leftReplicaFakeDb)
 
 	// During the 29th write, let the MASTER disappear.
+	masterDemoted := make(chan bool)
```

Comment:
If you keep this (see comment below), this should be `chan struct{}` because `bool` is not relevant here.